script.js

```diff
@@ -0,0 +1,208 @@
+// function show() {
+//   const x = document.getElementById('menubar');
+//   if (x.style.display === 'block' & x.className === 'navbar') {
+//     x.style.display = 'none';
+//     console.log('1');
+//   } else if (x.className === 'navbar') {
+//     x.className += ' active';
+//     console.log('2');
+
+//   } else {
+//     x.className = 'navbar';
+//     console.log('3');
+//   }
+// }
```

Comment:
- [ ] It's a good thing you currently load this part of your portfolio dynamically. How about removing the whole of this completely? Commenting old pieces of code in a workspace makes the code looks kinda messy :hammer_and_wrench: